lib/dealership.rb

```diff
@@ -0,0 +1,62 @@
+class Dealership
+  attr_reader :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    @inventory.size
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    @inventory != []
+  end
+
+  def cars_by_make(car_make)
+    @inventory.select do |car|
+      car_make == car.make
+    end
+  end
+
+  def total_value
+    total_cost_of_cars = 0
+    @inventory.map do |car|
```

Comment:
Since we aren't using the array being returned by the .map function, I would probably use .each here.